core/utils/useragent.ts

```diff
@@ -63,18 +59,16 @@ function has(name: string): boolean {
 
 // Browsers.  Logic from:
 // https://github.com/google/closure-library/blob/master/closure/goog/labs/useragent/browser.js
-isIe = has('Trident') || has('MSIE');
-isEdge = has('Edge');
 // Useragent for JavaFX:
 // Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.44
 //     (KHTML, like Gecko) JavaFX/8.0 Safari/537.44
 isJavaFx = has('JavaFX');
-isChrome = (has('Chrome') || has('CriOS')) && !isEdge;
+isChrome = (has('Chrome') || has('CriOS'));
```

Comment:
do we use these for anything? maybe we should fully get rid of the ones we don't need at all